src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -409,14 +410,19 @@ export default {
 			return this.$store.getters.getToken()
 		},
 
+		displayedHeadline() {
```

Comment:
```suggestion
		emptyContentLabel() {
```

Unlikely there would be a case for wording 'No matches found'. Even so, there is no need in `if {}` check inside the computed

Reply:
Doch, 'No conversations found' is shown `this.conversationsList.length === 0` but otherwise ( for filters, `this.conversationsList.length !== 0` ) , the wording 'No matches found' is applied

Reply:
Seems I messed up the comparison, want to show "match" when a filter is applied. So now picked the same logic as for the subline